fem/integ/bilininteg_mass_kernels.hpp

```diff
@@ -937,6 +1005,30 @@ void SmemPAMassApply3D_Element(const int e,
    MFEM_SYNC_THREAD;
 }
 
+inline void PAMassApply1D(const int NE,
```

Comment:
Same as above, both for possibly moving into `bilininteg_mass_kernels.cpp` and for adding a short comment for consistency: `// PA Mass Apply 1D kernel`.

Reply:
Done in 4438077f6